### PR TITLE
git-checkout: add example for checking out files across branches

### PR DESCRIPTION
**Pull request by sethwoodworth**

### CODE REVIEW

**Comment by waldyrious** [comment]
Thanks! Please use lowercase and snake_case for the tokens, per the guidelines in CONTRIBUTING.md

File: pages/common/git-checkout.md [hunk 1]
```diff
@@ -4,12 +4,16 @@
 
 - Switch to another branch:
```
Comment (agnivade) [inline pages/common/git-checkout.md]: suggestion - "Switch to an existing branch". To match with the next description.

File: pages/common/git-checkout.md [hunk 2]
```diff
@@ -4,12 +4,16 @@
...
 - Create and switch to a new branch:
 
-`git checkout -b {{BRANCH-NAME}}`
+`git checkout -b {{branch_name}}`
 
 - Undo unstaged local modification:
 
 `git checkout .`
+
+- Checkout a file from another branch:
```
Comment (agnivade) [inline pages/common/git-checkout.md]: I would refrain from using the word "checkout" as we are trying to explain what "checkout" means in the first place. How about - "Replace the contents of {{file_name}} in the current branch with {{branch_name}}" ? Or something on those lines ..

**Comment by agnivade** [comment]
@sethwoodworth - Any updates on this ?

File: pages/common/git-checkout.md (hunk 2 above)
Comment (waldyrious) [inline pages/common/git-checkout.md]: @agnivade I agree with your concern but that wording can be a little confusing IMO. How about: "Replace a file in the current working directory with the version of it committed in a given branch"?

**Comment by sethwoodworth** [comment]
@agnivade Apologies! I had a harddrive failure and lost track of this PR.  I've implemented your and @waldyrious' feedback and pushed a new, rebased commit.

**Comment by agnivade** [comment]
@sethwoodworth - oops, hard drive crashes are a pain. Hopefully, nothing too damaging.

Thanks for pushing the updates !

**Pull request merged**